src/components/PlayButton.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div id = "button">
+    <v-btn
+      color='teal'
+      dark
+      depressed
+      fab
+      @click="toggle"
+    >
+      <v-icon 
+        v-if="isPlaying"
+        large
+      >pause</v-icon>
+      <v-icon
+        v-else
+        large
+      >play_arrow</v-icon>
+    </v-btn>
+  </div>
+</template>
+<script>
+import { eventBus } from "../main";
+export default {
+  data () {
+    return {
+      isPlaying: false
+    }
+  },
+
+  methods: {
+    toggle () {
+      this.isPlaying = !this.isPlaying
+      isPlaying = this.isPlaying
+    }
+  },
+
+  created () {
+    setInterval(function(){
```

Comment:
I am not sure if this is a good way of making `PlayButton` and `TimelapseBar` talk to each other. We shouldn't be emitting this event every second our app is running. It should be emitted only when the button is toggled, since that's the only time when the state of the slider bar is required to change. Also, using a global variable ,`isPlaying`, that isn't part of our component's data but whose value depends directly on `this.isPlaying` is sort of confusing and unnecessary. I propose an alternate approach where the event is emitted only in `toggle()` method and `TimelapseBar` takes care of the rest. Have a look here for inspiration: https://stackoverflow.com/questions/47944557/using-vue-to-count-up-by-seconds A possible way to implement the answers on that thread for our needs would be to change `created()` in `TimelpaseBar`:
```javascript
created() {
  eventBus.$on("timelapse-toggle", (isPlaying) => { // respond to event triggered by toggling of the play button
    if (!isPlaying) {
      clearInterval(this.interval); // interval will be a data attribute of our component, initially set to null
    } else {
      this.interval = setInterval(this.moveSlider, 1000); // moveSlider will be a method of our component that handles the logic of updating this.sliderVal
    }
  })
}
```

Reply:
The state of the slide bar has to change in tandem with other things in various different parts of the app. The event is only emitted when the timelapse is playing; switching to an approach in which the interval is handled in the bar blocks other components from having access to it. The solution you've linked here seems to be a case in which there are only two components talking which is not going to be the case in our app. Creating multiple interval objects across several components seems much more complicated and error-prone than a single app-wide pulse.

Reply:
If your primary objection is to the pulse coming from the button component, it might make more sense to move it to TheTimelapse itself and have the button only emit something when it's toggled, but I firmly believe that one pulse for the whole app is the way to go.

Reply:
Let's talk about this in person tomorrow.